lib/signal_marker.cc

```diff
@@ -24,7 +22,7 @@
 #include "config.h"
 #endif
 
-#include <signal_marker.h>
+#include "signal_marker.h"
```

Comment:
Is this required? I'd thought it was correct with <>

Reply:
AFAIK quotation marks tell the compiler to look in the same directory for the header file. I guess that's right here?

Reply:
Should be right, yes, but it'd be also right to use <> when the _absolute_ path that follows the `INCLUDE_DIRS` is unambiguous. In this case, I assume, both conditions hold (as it was working in `master` branch). But that would mean that the change wasn't a _required_ one. 

It's maybe just pedantic. Lately, I'm more inclined to `<>` all the includes.

Reply:
I'd leave it as changed since all other `.cc` files use quotation marks to include their headers. This change was done to achieve consistency there.